tests/integration_tests/data_steward/validation/participants/validate_test.py

```diff
@@ -257,8 +236,11 @@ def test_identify_rdr_ehr_match(self):
                     (5, 8570, timestamp ('2000-11-01')),
                     (6, 0, timestamp ('1900-01-01')),
                     (7, 4215271, timestamp ('1981-01-10')),
+                    (8, 4214687, timestamp ('1999-12-1')),
                     (8, 4214687, timestamp ('1999-12-1'))
```

Comment:
That's a good question. That means a site is saying a participant has two birthdates. This is a problem they will need to fix, and will be shown via duplicates in the results.html report. For validation purposes, the deduplication query may choose one at random, so the result may not be accurate. This can also affect our unioned_ehr process. Ultimately, the site has to fix it.

Reply:
Added a commit for catching person duplicates